src/Microsoft.Health.Dicom.SqlServer/Features/Schema/SchemaVersion.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Health.Dicom.SqlServer.Features.Schema
     /// </summary>
     public enum SchemaVersion
     {
+        Unknown = 0,
```

Comment:
What's the rule that wanted this added?

Reply:
[CA1008](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1008)

The rule describes a desire to have `default(SchemaVersion)` or `default` to be a defined value of the enum where appropriate.

Reply:
I would err on the side of not adding the default here unless we know how this interacts with the schema migration/schema API changes.

Reply:
Reading the code there doesn't appear any usage that would interfere. I think we're ok to add this